examples/statistics/boxplot.py

```diff
@@ -67,28 +67,28 @@
                       markerfacecolor='firebrick')
 meanlineprops = dict(linestyle='--', linewidth=2.5, color='purple')
 
-fig, axes = plt.subplots(nrows=2, ncols=3, figsize=(6, 6), sharey=True)
-axes[0, 0].boxplot(data, boxprops=boxprops)
-axes[0, 0].set_title('Custom boxprops', fontsize=fs)
+fig, axs = plt.subplots(nrows=2, ncols=3, figsize=(6, 6), sharey=True)
+axs[0, 0].boxplot(data, boxprops=boxprops)
+axs[0, 0].set_title('Custom boxprops', fontsize=fs)
 
-axes[0, 1].boxplot(data, flierprops=flierprops, medianprops=medianprops)
-axes[0, 1].set_title('Custom medianprops\nand flierprops', fontsize=fs)
+axs[0, 1].boxplot(data, flierprops=flierprops, medianprops=medianprops)
+axs[0, 1].set_title('Custom medianprops\nand flierprops', fontsize=fs)
 
-axes[0, 2].boxplot(data, whis='range')
-axes[0, 2].set_title('whis="range"', fontsize=fs)
+axs[0, 2].boxplot(data, whis='range')
+axs[0, 2].set_title('whis="range"', fontsize=fs)
 
-axes[1, 0].boxplot(data, meanprops=meanpointprops, meanline=False,
+axs[1, 0].boxplot(data, meanprops=meanpointprops, meanline=False,
```

Comment:
more alignment... (and below as well)
perhaps you can temporarily remove the flake8 rule excluding the check for these bad alignments to find all these cases locally :p

Reply:
Done. Of course that was the last missing indent - well there are still lots, but not induced by this PR.